lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -422,13 +425,17 @@ void _testContainer() {
     updateNode(
       builder,
       id: 0,
-      actions: 0,
-      flags: 0,
       transform: Matrix4.identity().toFloat64(),
       rect: zeroOffsetRect,
       childrenInHitTestOrder: Int32List.fromList(<int>[1]),
       childrenInTraversalOrder: Int32List.fromList(<int>[1]),
     );
+    updateNode(
```

Comment:
Maybe add a comment about why this additional update is needed

Reply:
It was just a bug in the previous code. It pretended to have a child, but didn't send child info. I added more asserts in the `SemanticsOwner`, which uncovered the bug. I don't think a comment is necessary because this is now enforced everywhere (I'd have to write many comments like that, everywhere there are child nodes), and the assertion failure should make it obvious what's wrong if someone mistakenly removes it.

Reply:
sg